internal/envoy/handler.go

```diff
@@ -91,7 +91,7 @@ func (r *RequestHandler) OnStreamRequest(streamID int64, req *discovery.Discover
 		r.logger.Error("error fetching gateway", "error", err)
 		return err
 	}
-	allowed, err := gateway.CanFetchSecrets(ctx, resources)
+	allowed, err := gateway.CanFetchSecrets(resources)
```

Comment:
You'll see lots of changes like this just removing the previously-unused `context.Context` parameter